keyboards/handwired/split89/info.json

```diff
@@ -10,7 +10,97 @@
+                    {"label":"PgUp" ,  "matrix": [7, 9], "x":20 ,  "y":1.5},
+                    {"label":"Tab" ,  "matrix": [3, 4], "x":0 ,  "y":2.5 ,  "w":1.5},
+                    {"label":"Q" ,  "matrix": [2, 5], "x":1.5 ,  "y":2.5},
+                    {"label":"W" ,  "matrix": [2, 6], "x":2.5 ,  "y":2.5},
+                    {"label":"E" ,  "matrix": [2, 7], "x":3.5 ,  "y":2.5},
+                    {"label":"R" ,  "matrix": [2, 8], "x":4.5 ,  "y":2.5},
+                    {"label":"T" ,  "matrix": [2, 9], "x":5.5 ,  "y":2.5},
+                    {"label":"Y" ,  "matrix": [9, 0], "x":9.25 ,  "y":3.5},
+                    {"label":"U" ,  "matrix": [8, 0], "x":10.25 ,  "y":2.5},
+                    {"label":"I" ,  "matrix": [8, 1], "x":11.25 ,  "y":2.5},
+                    {"label":"O" ,  "matrix": [8, 2], "x":12.25 ,  "y":2.5},
+                    {"label":"P" ,  "matrix": [8, 3], "x":13.25 ,  "y":2.5},
+                    {"label":"{" ,  "matrix": [8, 4], "x":14.25 ,  "y":2.5},
+                    {"label":"}" ,  "matrix": [8, 5], "x":15.25 ,  "y":2.5},
+                    {"label":"|" ,  "matrix": [8, 6], "x":16.25 ,  "y":2.5 ,  "w":1.5},
+                    {"label":"Delete" ,  "matrix": [8, 7], "x":18 ,  "y":2.5},
+                    {"label":"End" ,  "matrix": [8, 8], "x":19 ,  "y":2.5},
+                    {"label":"PgDn" ,  "matrix": [8, 9], "x":20 ,  "y":2.5},
+                    {"label":"Caps Lock" ,  "matrix": [4, 4], "x":0 ,  "y":3.5 ,  "w":1.75},
+                    {"label":"A" ,  "matrix": [3, 5], "x":1.75 ,  "y":3.5},
+                    {"label":"S" ,  "matrix": [3, 6], "x":2.75 ,  "y":3.5},
+                    {"label":"D" ,  "matrix": [3, 7], "x":3.75 ,  "y":3.5},
+                    {"label":"F" ,  "matrix": [3, 8], "x":4.75 ,  "y":3.5},
+                    {"label":"G" ,  "matrix": [3, 9], "x":5.75 ,  "y":3.5},
+                    {"label":"H" ,  "matrix": [10, 0], "x":9.5 ,  "y":4.5},
+                    {"label":"J" ,  "matrix": [9, 1], "x":10.5 ,  "y":3.5},
+                    {"label":"K" ,  "matrix": [9, 2], "x":11.5 ,  "y":3.5},
+                    {"label":"L" ,  "matrix": [9, 3], "x":12.5 ,  "y":3.5},
+                    {"label":":" ,  "matrix": [9, 4], "x":13.5 ,  "y":3.5},
+                    {"label":"\"" ,  "matrix": [9, 5], "x":14.5 ,  "y":3.5},
+                    {"label":"Enter" ,  "matrix": [9, 6], "x":15.5 ,  "y":3.5 ,  "w":2.25},
+                    {"label":"Shift" ,  "matrix": [5, 4], "x":0 ,  "y":4.5 ,  "w":2.25},
+                    {"label":"Z" ,  "matrix": [4, 5], "x":2.25 ,  "y":4.5},
+                    {"label":"X" ,  "matrix": [4, 6], "x":3.25 ,  "y":4.5},
+                    {"label":"C" ,  "matrix": [4, 7], "x":4.25 ,  "y":4.5},
+                    {"label":"V" ,  "matrix": [4, 8], "x":5.25 ,  "y":4.5},
+                    {"label":"B" ,  "matrix": [4, 9], "x":6.25 ,  "y":4.5},
+                    {"label":"N" ,  "matrix": [11, 0], "x":10 ,  "y":5.5},
```

Comment:
```suggestion
                    {"label":"N" ,  "matrix": [11, 0], "x":10 ,  "y":4.5},
```

Reply:
This has an image of it for reference, if needed.

https://github.com/jurassic73/split89